src/components/login/LoginForm.js

```diff
@@ -29,13 +31,20 @@ const LoginForm = (props) => {
                     value ={password}
                     onChange={passwordChange}
                 />
-                <label className="active" for="password">Password</label>
+                <label>Password</label>
```

Comment:
@akram256, please fix this to `<label className="active" for="password">Password</label>` it's causing this issue 
![image](https://user-images.githubusercontent.com/28726900/56098698-83addf80-5f0b-11e9-9c74-57670bdb2dc3.png)

Reply:
worked on it